.github/workflows/node-ci-push.yml

```diff
@@ -137,3 +137,53 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
         projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
         autoAcceptChanges: true
+
+  publish_to_branch:
```

Comment:
This is pretty much a copy from the render-gateway, right, except for the repo being cloned?

I wonder if it is worth annotating that. If we end up doing some shared workflows it might be useful to know.